components/builder-api/src/server/resources/channels.rs

```diff
@@ -73,6 +73,8 @@ impl Channels {
                   web::delete().to(delete_channel))
            .route("/depot/channels/{origin}/{channel}/pkgs",
                   web::get().to(get_packages_for_origin_channel))
+           .route("/depot/channels/{origin}/{channel}/pkgs/latest",
```

Comment:
More and more I'm wondering if .route("/depot/channels/{origin}/{channel}/latest" makes more sense to avoid conflicts with a package named 'latest'.

Reply:
It's a philosophical question to be sure. It might make sense for us to namespace out `latest` as an unusable package name. The only concern with refactoring the route would be that it breaks slightly away from the established pattern but, that doesn't matter very much I don't think.  It might be the right thing to change it.

Reply:
A possible approach to retain clarity that `pkgs` are being queried would be to borrow from the REST patterns used by Elasticsearch endpoints, many of which use `_`, for instance:
`-XGET localhost:9200/_snapshot/_status?pretty`.

It could translate into: `/depot/channels/{origin}/{channel}/pkgs/_latest`. We could make a reasonable rule to disallow uploads of packages with beginning with `_` and also retro fit all the current relevant endpoints to accept `_latest` as a synonym for `/latest`.